manifest/manifest.go

```diff
@@ -12,6 +12,39 @@ import (
 	cbg "github.com/whyrusleeping/cbor-gen"
 )
 
+const (
+	AccountKey  = "account"
+	CronKey     = "cron"
+	DataCapKey  = "datacap"
+	InitKey     = "init"
+	MarketKey   = "storagemarket"
+	MinerKey    = "storageminer"
+	MultisigKey = "multisig"
+	PaychKey    = "paymentchannel"
+	PowerKey    = "storagepower"
+	RewardKey   = "reward"
+	SystemKey   = "system"
+	VerifregKey = "verifiedregistry"
+)
+
+func GetBuiltinActorsKeys() []string {
+	keys := []string{
+		AccountKey,
+		CronKey,
+		DataCapKey,
+		InitKey,
+		MarketKey,
+		MinerKey,
+		MultisigKey,
+		PaychKey,
+		PowerKey,
+		RewardKey,
+		SystemKey,
+		VerifregKey,
+	}
+	return keys
+}
```

Comment:
We have a version on this in `chain/actors/manifest.go`. I think this is a better location for this, but it still should take actors version as an input.